static/app/utils/profiling/canvasView.tsx

```diff
@@ -44,7 +44,7 @@ export class CanvasView<T extends {configSpace: Rect}> {
     this.model = model;
     this.canvas = canvas;
     this.depthOffset = options.depthOffset ?? 0;
-    this.barHeight = options.barHeight ? options.barHeight * window.devicePixelRatio : 0;
```

Comment:
This was causing undefined behavior when we initialize the view as we were dividing by 0